oaknational/Oak-Web-Application
# Refactor remove search code from unit and lesson list LESQ-461

## Description

-Removes search related code from unit and lesson lists
-Move tracking events up to page level

Music year: 1958


## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2126--oak-web-application.netlify.thenational.academy
Unit and lesson lists should look the same
Tracking events are being send when a item is clicked



## Checklist
